zerver/webhooks/sonarr/view.py

```diff
@@ -0,0 +1,105 @@
+from typing import Any, Dict
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import REQ, has_request_variables, webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+SONARR_TOPIC_TEMPLATE = "{series_title}".strip()
+SONARR_TOPIC_TEMPLATE_TEST = "Sonarr - Test".strip()
+SONARR_TOPIC_TEMPLATE_HEALTH_CHECK = "Health {level}".strip()
+
+SONARR_MESSAGE_TEMPLATE_TEST = "Your Sonarr webhook has been confgured correctly."
+SONARR_MESSAGE_TEMPLATE_SERIES_DELETED = "{series_title} has been deleted.".strip()
+SONARR_MESSAGE_TEMPLATE_HEALTH_CHECK = "{message}.".strip()
+SONARR_MESSAGE_TEMPLATE_EPISODES_RENAMED = "{series_title} episodes have been renamed.".strip()
+SONARR_MESSAGE_TEMPLATE_EPISODE_IMPORTED = (
+    "{series_title} - {series_number}x{episode_number} - {episode_name} has been imported.".strip()
+)
+SONARR_MESSAGE_TEMPLATE_EPISODE_IMPORTED_UPGRADE = "{series_title} - {series_number}x{episode_number} - {episode_name} has been upgraded to {new_quality} from {old_quality}.".strip()
```

Comment:
I would recommend phrasing this template using the from/to sequence instead of to/from.